capstone/src/main/java/com/google/univiz/scorecard/CollegeIdReaderProviderImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.google.univiz.scorecard;
+
+import com.google.univiz.api.CollegeId;
+import com.google.univiz.config.UnivizConfig;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import javax.inject.Inject;
+
+class CollegeIdReaderProviderImpl implements CollegeIdReaderProvider {
+  private final UnivizConfig univizConfig;
+
+  @Inject
+  protected CollegeIdReaderProviderImpl(UnivizConfig univizConfig) {
+    this.univizConfig = univizConfig;
+  }
+
+  @Override
+  public Reader getReaderFromCollegeIds(List<CollegeId> ids) throws IOException {
+    String urlString = "https://api.data.gov/ed/collegescorecard/v1/schools.json?id=";
+    for (CollegeId id : ids) {
+      urlString += Integer.toString(id.id());
+      urlString += ",";
+    }
```

Comment:
Have a look at [StringJoiner](https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html) to simplify this logic.